tests/pos/typeclass-encoding3b.scala

```diff
@@ -345,5 +345,12 @@ object functors {
   }
 
   MonadFlatten.flattened(List(List(1, 2, 3), List(4, 5))) // ok, synthesizes (using ListMonad)
-  MonadFlatten.flattened(List(List(1, 2, 3), List(4, 5)))(using ListMonad) // error
+  MonadFlatten.flattened(List(List(1, 2, 3), List(4, 5)))(using ListMonad) // was an error
+  /*
+  When checking `ListMonad <:< functors.Monad.Impl[T]`
+  we eventually get to the comparison `[X] =>> T[X] <:< [+X] =>> List[X]`
+  because the `This` type member of `ListMonad` has a covariance annotation.
+  This fails the variance conformance checks despite the fact that T has been instantiated to List,
```

Comment:
I don't follow the comment.  How does it failing the conformance check lead it to not erroring?

Reply:
Ah, I'd overlooked updating the tense